### ISSUE TITLE
New ZIM request: Runarcana RPG

### ISSUE THREAD

**Issue by arddhu-hub** [openzim/zim-requests#408]
- Website URL: https://wiki.runarcana.org/
- License: https://www.riotgames.com/en/legal
- Desired ZIM Title: Runarcana
- Desired ZIM Description: Runarcanana RPG Wiki
- Desired ZIM Icon –png (URL or attach one): https://wiki.runarcana.org/images/logo.png?ebd3d
- Language (ISO 639-3): brazilian portuguese
- Desired Main Page (homepage, if different from website URL):  homepage
- Is this a MediaWiki?: yes

**Comment by Popolechien** [comment]
I've made a query with Riot to confirm this was ok and I've also started the recipe here: https://farm.openzim.org/recipes/runarcana_pt_all

We'll see how mwoffliner handles it.

**Comment by arddhu-hub** [comment]
Ok, but this is not a Riot Games project, it is a fanmade content with Riot
licensing.

Em qui., 27 de jan. de 2022 às 10:29, Popo le Chien <
***@***.***> escreveu:

**Comment by Popolechien** [comment]
Yep, I went through their legalese part and that should be ok. However we still will be copying their trademarks and visuals since the zim is an integral copy. Always good practice to be on their radar.

**Comment by arddhu-hub** [comment]
I am the author of the wiki and its contents

Em qui, 27 de jan de 2022 10:46, Popo le Chien ***@***.***>
escreveu:

**Comment by Popolechien** [comment]
Ah my bad, I thought there'd be some proprietary artwork - had a browse through and we're good. Nice work.

Now for the bad news: mwoffliner did not catch pages that were under the Special: namespace, sorry. I'll park the issue for the time being and we'll figure it out when mwoffliner is being revamped, that's actually a pretty good use case.

**Comment by arddhu-hub** [comment]
I was about to ask this, the way the wiki is configured is to use as
special pages with language redirection, and I see that it don't work with
kiwi like that.

Em qui., 27 de jan. de 2022 às 11:02, Popo le Chien <
***@***.***> escreveu:

**Comment by benoit74** [comment]
We need to adapt scraper to support Minerva skin: https://github.com/openzim/mwoffliner/issues/2343
